src/server/ua_securechannel_manager.c

```diff
@@ -150,24 +259,28 @@ UA_SecureChannelManager_renew(UA_SecureChannelManager *cm, UA_Connection *conn,
     return UA_STATUSCODE_GOOD;
 }
 
-UA_SecureChannel *
-UA_SecureChannelManager_get(UA_SecureChannelManager *cm, UA_UInt32 channelId) {
-    channel_list_entry *entry;
-    LIST_FOREACH(entry, &cm->channels, pointers) {
-        if(entry->channel.securityToken.channelId == channelId)
+UA_SecureChannel*
+UA_SecureChannelManager_get(UA_SecureChannelManager* cm, UA_UInt32 channelId)
+{
+    channel_list_entry* entry;
+    LIST_FOREACH(entry, &cm->channels, pointers)
+    {
+        if (entry->channel.securityToken.channelId == channelId)
             return &entry->channel;
     }
     return NULL;
 }
 
 UA_StatusCode
-UA_SecureChannelManager_close(UA_SecureChannelManager *cm, UA_UInt32 channelId) {
-    channel_list_entry *entry;
-    LIST_FOREACH(entry, &cm->channels, pointers) {
-        if(entry->channel.securityToken.channelId == channelId)
+UA_SecureChannelManager_close(UA_SecureChannelManager* cm, UA_UInt32 channelId)
+{
+    channel_list_entry* entry;
+    LIST_FOREACH(entry, &cm->channels, pointers)
+    {
+        if (entry->channel.securityToken.channelId == channelId)
             break;
     }
-    if(!entry)
+    if (!entry)
```

Comment:
I will revert those changes, no problem.
I used my own coding style here without checking first.
Same for the brackets on new lines.